src/Voron/Data/Tables/Table.cs

```diff
@@ -1467,6 +1498,56 @@ public void PrepareForCommit()
             }
         }
 
+        private void ThrowInconsistentItemsCountInIndexes(string indexName, long expectedSize, long actualSize)
+        {
+            throw new InvalidOperationException($"Inconsistent index items count detected! Index name: {indexName} expected size: {expectedSize} actual size: {actualSize}");
+        }
+
+        /// <summary>
+        /// validate all globals indexes has the same number
+        /// validate all local indexes has the same number as the table itself
+        /// </summary>
+        internal void AssertValidTable()
```

Comment:
This needs to be a conditional method using `VALIDATE`

Reply:
Thee is one overload for that purpose: https://github.com/maximburyak/ravendb/blob/dba853067d32c8a36a525906afcc2860e20a7855/src/Voron/Data/Fixed/FixedSizeTree.cs#L327

I've created another function, to allow fst validations, during our schema validation code.
And they share the same function to prevent code duplication.

Reply:
Yes, but you had 3 of them `ForceValidateTree`, `ValidateTree` and `AssertTreeIsValid`

Reply:
Yeah, I could keep only two of them